plugins/API/lang/en.json

```diff
@@ -1,18 +1,18 @@
 {
     "API": {
         "ChangeTokenHint": "If you want to change this token, please go to your %1$spersonal settings page%2$s.",
-        "GenerateVisits": "If you don't have data for today you can first generate some data using the %1$s plugin. You can enable the %2$s plugin, then click on the 'Visitor Generator' menu in the Matomo Admin area.",
+        "GenerateVisits": "If you don't have data for today, you can first generate some data using the %1$s plugin. You can turn on the %2$s plugin, then click the 'Visitor Generator' menu in the Matomo-admin area.",
```

Comment:
I don't know if Matomo-admin is better than Matomo Admin - is it wording you've found somewhere else or is there some rationale here? Other than this question these changes look good.

Reply:
It isn't a admin area for Matomo, but rather an area for the Matomo admins(?) The distinction isn't great, but that seems like the point of going with "A", which I think is an error.

Is it "the area for Matomo admins" or "the area for administration of Matomo"?

Reply:
```suggestion
        "GenerateVisits": "If you don't have data for today, you can first generate some data using the %1$s plugin. You can turn on the %2$s plugin, then click the 'Visitor Generator' menu in the Matomo admin-area.",
```

Reply:
Equally we could make it even more explicit by saying something like click 'Development > Visitor Generator' in the Matomo Administration area.